content/ja/handbook/engineering/infrastructure-platforms/developer-experience/index.md

```
@@ -0,0 +1,183 @@
+DevEx は、進捗状況の追跡と新しいロードマップ項目の特定のために四半期ごとの DX サーベイを実施します。
+
+## DevEx における AI
+
+DevEx チームは AI ツールを積極的に活用して自分たちの作業を加速させ、内側から GitLab の AI 機能を検証しています。
+
+- [DevEx における AI の活用方法](ai/) — ツール、ガイドライン、ワークフロー、ドッグフーディングのプラクティス
+- [トップヒント](ai/top-tips/) — 日々のエンジニアリング業務のための実践的な AI ワークフロー
+
+## 私たちとの協働
+
+各 [DevEx チーム](#team-structure) はハンドブックページにロードマップを維持し、トップレベルのチームエピックにリンクされたエピックに取り組みます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)